src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
```

Comment:
better to use Colors enum here and everywhere in class
```suggestion
    Colors ballColor;
```